go/libkb/keyfamily.go

```diff
@@ -576,6 +576,13 @@ func (ckf *ComputedKeyFamily) Revoke(tcl TypedChainLink) (err error) {
 
 // SetPGPHash sets the authoritative version (by hash) of a PGP key
 func (ckf *ComputedKeyFamily) SetActivePGPHash(kid keybase1.KID, hash string) {
+	found := false
+	if ks, ok := ckf.kf.PGPKeySets[kid]; ok && ks != nil && ks.KeysByHash[hash] != nil {
+		found = true
+	}
+	if !found {
+		ckf.G().Log.Warning("Didn't have a PGP key for %s with hash %s", kid, hash)
```

Comment:
Maybe add a comment here clarifying that this condition is unexpected and likely to cause big errors elsewhere?